lms/djangoapps/grades/rest_api/v1/tests/test_gradebook_views.py

```diff
@@ -1520,16 +1537,24 @@ def test_with_override_with_history(self, login_method):
             ]),
             'course_id': text_type(self.course_key),
             'subsection_id': text_type(self.usage_key),
-            'history': [{
-                'user': self.global_staff.username,
-                'comments': None,
-                'created': '2019-01-01T00:00:00Z',
-                'feature': 'GRADEBOOK',
-                'action': 'CREATEORUPDATE'
-            }]
+            'history': [OrderedDict([
+                ('created', created_date),
+                ('grade_id', 1),
+                ('history_id', 1),
+                ('earned_all_override', 0.0),
+                ('earned_graded_override', 0.0),
+                ('history_change_reason', 'GRADEBOOK'),
+                ('history_date', history_date),
+                ('history_type', u'+'),
+                ('history_user', None),
```

Comment:
Hmm, I'm surprised that this is null, especially since we're providing a `requesting_user` parameter in the `update_or_create_override` call.